src/opentelemetry.erl

```diff
@@ -119,8 +118,6 @@
 %% Note that this also excludes tabs, newlines, carriage returns, etc.
 -type tracestate()         :: [{unicode:latin1_chardata(), unicode:latin1_chardata()}].
 
--type stack_trace()        :: [erlang:stack_item()].
```

Comment:
these appeared unused, and is also being deprecated in OTP 24

Reply:
Weird, what is it being replaced with in OTP 24?